hadoop-hdds/interface-server/src/main/proto/ScmServerProtocol.proto

```diff
@@ -48,6 +48,7 @@ message SCMBlockLocationRequest {
   optional string traceID = 2;
 
   optional UserInfo userInfo = 3;
+  optional uint32 version = 4;
```

Comment:
Will clientVersion be more straightforward than version in these protos,  in case we might add other version into proto message ?

Reply:
They are the same.